### ISSUE TITLE
Mac-V0010-QA1 - MACOS security/Exceptionally long initial times.

### ISSUE THREAD

**Issue by jgbr** [dapperrobotbear/Stage9#210]
As you are probably aware, the  DMG upsets Mac OS 'App Store and identified developers security warning' resulting in users having to head to the control panel and authorise it. I don't think you can get around this without paying apple money to be a dev or something.

The DMG/APP then takes a huge amount of time to open and execute although little impact on system resources is taking place.

I am running this on a i7 3.1Ghz HQ CPU and 16GB ram with Radeon Pro 560 4B Vram.

I have an app called istat which showed no massive tax on the system or memory. Either it's just the process of loading it that takes time or that security trip might have something to do with it.

Either way, ensuring in documentation this is mention will probably save some hassle down the line.

**Comment by jgbr** [comment]
On a second run it is quicker.

**Comment by dapperrobotbear** [comment]
The file is a ZIP file, and not a DMG. Decompressing it (since it's 5GB) does take time. Depending on your machine specs, your performance may vary.

**Comment by Jaredsfs1** [comment]
Confirmed, but it's not a bug, it's just MacOS's handling of a large file combined with it's security checks.  You can speed it up by disabling Gatekeeper in System Preferences.

**Comment by jgbr** [comment]
I don't think we should be recommending users disable OSX security features. It could open Stage 9 up to 'problems'. Instead best to advise in install documentation, about process of authorising.

**Comment by dapperrobotbear** [comment]
Opening non-signed apps is documented by the gatekeeper process itself. Like: https://support.apple.com/library/content/dam/edam/applecare/images/en_US/osx/exemption_dialog.png

**Comment by dapperrobotbear** [comment]
That's OSX security. 😄 Since this isn't necessarily a bug with Stage9 I'm going to close this issue.
